version_mapping/3.1.0-4.0.0/blacklist-shapes-400.ttl

```diff
@@ -22,7 +22,7 @@ shapes:
 
 shapes:HostShape
 	a sh:NodeShape ;
-	sh:targetClass ids:Host ;
+	sh:targetClass ids:Endpoint ;
```

Comment:
```suggestion
	sh:targetClass ids:Host ;
```

It was actually Host in this previous version, so you must not change that here. Please revert, thank you.